.github/workflows/release.yml

```diff
@@ -138,3 +138,18 @@ jobs:
         tag: ${{ github.ref }}
         file: /tmp/anycloud-macos.tar.gz
         asset_name: anycloud-macos.tar.gz
+
+    - name: Update homebrew
+      # TODO: should we create a new token or reuse the AUTOPUBLISH one?
+      run: |
+        alanversion=$(echo ${{github.ref}} | sed "s/refs\/tags\///")
+        [[ $alanversion != *'beta'* ]] && \
```

Comment:
Is this syntax right? The stars on the outside of the single quotes is so weird.

Reply:
Yes, it is bash partial matching